src/nvim/version.c

```diff
@@ -2834,24 +2836,39 @@ void intro_message(int colon)
     for (i = 0; i < (int)ARRAY_SIZE(lines); i++) {
       p = lines[i];
 
+      if (strstr(p, "news") != NULL) {
+        p = _(p);
+        mesg_size = snprintf(NULL, 0, p,
+                             STR(NVIM_VERSION_MAJOR), STR(NVIM_VERSION_MINOR));
+        assert(mesg_size > 0);
+        mesg = xmallocz((size_t)mesg_size);
+        snprintf(mesg, mesg_size + 1, p,
+                 STR(NVIM_VERSION_MAJOR), STR(NVIM_VERSION_MINOR));
+      }
+
       if (sponsor != 0) {
         if (strstr(p, "children") != NULL) {
-          p = sponsor < 0
+          mesg = sponsor < 0
               ? N_("Sponsor Vim development!")
               : N_("Become a registered Vim user!");
-        } else if (strstr(p, "iccf") != NULL) {
-          p = sponsor < 0
+          p = NULL;
+        }
+        if (strstr(p, "iccf") != NULL) {
+          mesg = sponsor < 0
               ? N_("type  :help sponsor<Enter>    for information ")
               : N_("type  :help register<Enter>   for information ");
-        } else if (strstr(p, "Orphans") != NULL) {
-          p = N_("menu  Help->Sponsor/Register  for information    ");
+          p = NULL;
         }
       }
 
-      if (*p != NUL) {
-        do_intro_line(row, _(p), 0);
+      if (*mesg != NUL) {
+        do_intro_line(row, mesg, 0);
```

Comment:
This doesn't look right. With this change the `:help news` line is translated, but the other lines are not.